server_addon/houdini/server/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.1.5"
+__version__ = "0.1.5-spmake.1"
```

Comment:
You don't appear to be making any server changes so you don't need to increment houdini server version as far as I know. Also, make sure to keep the PR to match Ayon versioning instead of your studio one of course.
```suggestion
__version__ = "0.1.5"
```